### PR TITLE
Rearrange the code of initialUI.py

### PR DESCRIPTION
**Pull request by 0ranjuice**
Combine initial page with loading animation and restructure the entire Python Flet code for extensibility and readability

### CODE REVIEW

File: initialUI.py [hunk 1]
```diff
@@ -22,72 +22,130 @@
...
-    pr = ProgressRing(width=35, height=35, stroke_width=10)
-    for i in range(0, 101):
-        pr.value = i * 0.01
-        sleep(0.05)
-        page.update()
+    # add application's root control to the page here
+    page.add(app)
 
 
 flet.app(target=main, assets_dir=f"assets")
```
Comment (DeChickenKing) [inline initialUI.py:151]:
```suggestion
flet.app(target=main, assets_dir=f"assets")

```
pep8 endline

File: initialUI.py [hunk 2]
```diff
@@ -22,72 +22,130 @@
...
+                self.connection_status,
+                self.btn_pair
+            ],
+            alignment='center',
+            horizontal_alignment='center',
+        )
+
+        return self.rtn_display
+
+    def btn_pair_clicked(self, e):
```
Comment (DeChickenKing) [inline initialUI.py:91]:
```suggestion
    def button_pair_clicked(self, e):
```
the more descriptive the better

File: initialUI.py [hunk 3]
```diff
@@ -22,72 +22,130 @@
...
+        self.rtn_display.controls.pop()
+        self.connection_status.content.value = 'Connected'
+        self.connection_status.content.color = colors.GREEN_300
+        self.update()
+
+    def update(self):
+        super().update()
+
+
+class DropdownMenu(UserControl):
```
Comment (DeChickenKing) [inline initialUI.py]:
```suggestion
class StationsDropdownMenu(UserControl):
```
same as above

File: initialUI.py (hunk 2 above)
Comment (0ranjuice) [reply initialUI.py:91]: Fuck u ! NO!

**Review by DeChickenKing — APPROVED** [review]
fixed shit, now merge it

**Review by foreverchankang — APPROVED** [review]

**Pull request merged**